elegant/worm_spline.py

```diff
@@ -37,6 +37,8 @@ def pose_from_mask(mask, smoothing=2):
     sx, sy = slices[0]
     cropped = mask[sx, sy]
     centerline, widths = _get_centerline(cropped)
+    if centerline.size == 0:
```

Comment:
I think it will probably blow up if we have a centerline less than 3 pixels (or maybe even more). Proposal: require `len(centerline) > 10` here.

Reply:
Sounds reasonable.